site/go/main.go

```diff
@@ -30,7 +30,7 @@ func main() {
 	dsEmulatorHost := flag.String("ds_emu_host", "", "The emulator host address for cloud datastore")
 	dsProjectId := flag.String("ds_emu_project_id", "", "The gcp project id for the datastore emulator")
 	ipAddressConfig := flag.String("ip_address", "0.0.0.0", "IP address of server")
-	portConfig := flag.String("port", "8080", "Port to serve application")
+	portConfig := flag.String("port", "8081", "Port to serve application")
```

Comment:
You've just changed the default port to 8081? I thought the expected behavior was to check if it's taken and increment if it is